tests/testthat/test-crayonex.R

```diff
@@ -527,3 +527,18 @@ test_that("ansi_strtrim", {
 
   for (case in cases) expect_equal(ansi_strtrim(case[[1]], 10), case[[2]])
 })
+
+test_that("ansi_columns", {
+  withr::local_options(c(cli.unicode = FALSE))
+  local_edition(3)
```

Comment:
Note that this is new in `testthat` 3, and so `DESCRIPTION` needs to be updated to set that as the required minimum.

Reply:
Right. OTOH R does not enforce version requirements of suggested packages, so we would need to check for this ourselves in `testthat.R` and error there. So we would trade this error to another one.